### DISCUSSION TITLE
我有一个TERMINAL的费用一直在计算如何取消

### DISCUSSION THREAD

**Discussion by Xishang7707** [labring/sealos#4565] [Q&A]
![image](https://github.com/labring/sealos/assets/38380429/425bd6c1-dc3f-4a19-9092-fca6e5c7289d)
之前从模板中装过一个应用然后已经删除了，也打开过控制台操作过，那个应用费用已经没有了，但是控制台一直在，不知道如何关闭

**Comment by bxy4543** [comment]
这个是桌面终端的费用，长时间不使用的话 会自己关闭
